### ISSUE TITLE
Specify protocols in use

### ISSUE THREAD

**Issue by ismogis** [oskariorg/oskari-documentation#57]
It seems that all traffic between the frontend and backend uses the HTTP protocol. However, I couldn't find this explicitly mentioned in the documentation. This information is essential when designing server infrastructure and selecting appropriate software.

A logical place to include this detail might be the graph under section 2.2.1 Frontend Architecture, or perhaps just a brief mention in 1.1.1 Technical Overview.

**Comment by ZakarFin** [comment]
Yes, previously we used websockets as well, but they are no longer used. Though there still might be some page mentioning them regarding how WFS-layers work.
